src/main/java/study/racing/domain/Cars.java

```diff
@@ -0,0 +1,66 @@
+package study.racing.domain;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Cars {
+
+    private final List<Car> cars;
+
+    public Cars(List<CarName> names) {
+        this.cars = initCars(names);
+    }
+
+    public Cars(String... names) {
+        this.cars = initCars(Arrays.stream(names).map(CarName::new).toList());
+    }
+
+    public Cars(Car... cars) {
+        this.cars = List.of(cars);
+    }
+
```

Comment:
```suggestion
    public Cars(List<Car> cars) {
        this.cars = cars;
    }
```

주 생성자를 통해 필드 값을 초기화하고, 부 생성자는 주 생성자를 통해 필드 값을 초기화하도록 구현한다.